PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -25,6 +25,8 @@
         'keep recoCaloClusters_reducedEgamma_*_*',
         'keep EcalRecHitsSorted_reducedEgamma_*_*',
 
+        'keep *_egmPhotonPUPPIIsolationForPhotons_*_*', 
```

Comment:
is it needed in miniAOD?
why not just save them in the photon itself?
I'm not sure if these will remain readable, since these valueMaps are keyed off the selectedPatPhotons (not saved) while only  slimmedPhotons are saved.